packages/api-gateway/src/routers/apiGatewayRouter.ts

```diff
@@ -466,14 +466,14 @@ const apiGatewayRouter = (
             ctx,
             req.params.purposeId
           );
-          return res.status(200).json(agreement).send();
+          return res.status(200).send(agreement);
         } catch (error) {
           const errorRes = makeApiProblem(
             error,
             getAgreementByPurposeErrorMapper,
             ctx.logger
           );
-          return res.status(errorRes.status).json(errorRes).end();
+          return res.status(errorRes.status).send(errorRes);
         }
       }
     )
```

Comment:
There are other calls returning `.json` in this file, the `/organizations/...` ones